node-gui/src/main_window/main_widget/tabs/wallet/mod.rs

```diff
@@ -360,6 +361,7 @@ impl WalletTab {
                 self.account_state.send_delegation_id.clear();
                 Command::none()
             }
+            WalletMessage::NoOp => Command::none(),
```

Comment:
Why is this needed? What's the difference between sending a NoOp and not sending anything?

Reply:
There are cases where I have to send a message, like in closures with input validation: Fn(String) -> Message. So NoOp is a workaround there

Reply:
Ok. I don't have a strong opinion about this.